### PR TITLE
Add more Utopia credentials

### PR DESCRIPTION
**Pull request by BigBlueHat**
- Add .editorconfig.
- Add Utopia DL, EAD, and NatCert.

### CODE REVIEW

**Review by tminard — APPROVED** [review]

**Review by msporny — APPROVED** [review]

**Review by jameseaster — APPROVED** [review]

File: credentials/utopia-ead/landscape-back.svg
Comment (davidlehn) [inline credentials/utopia-ead/landscape-back.svg:1]: - The "Drivers License" text here is a typo.
- The demo printed cards had a lighter/faded background vs this.  That made the MRZ background more white.  (Or at least that's how it shows up in my browser here.)

Comment (BigBlueHat) [reply credentials/utopia-ead/landscape-back.svg:1]: Good catch. This is also using [OCR-A](https://en.wikipedia.org/wiki/OCR-A), but we found that [OCR-B](https://en.wikipedia.org/wiki/OCR-B) was much easier/faster to scan these days...so I'll change that also.

Comment (BigBlueHat) [reply credentials/utopia-ead/landscape-back.svg:1]: And actually...we probably don't want those on-card instructions there long term, so I'll remove those also (from this and the EAD).

Comment (BigBlueHat) [reply credentials/utopia-ead/landscape-back.svg:1]: Fixed in 77fcb82e3d4d4fc85562894aa2e1f092910d8b31

File: credentials/utopia-ead/landscape-front.svg
Comment (davidlehn) [inline credentials/utopia-ead/landscape-front.svg:1]: Not sure how this will be used, but the back has rounded corners like a card, the front does not.

**Review by davidlehn — APPROVED** [review]
Needs a changelog entry.

**Comment by BigBlueHat** [comment]
> Needs a changelog entry.

Changelog entry added along with making all the SVGs "match" in terms of rounded corners. I also found some of them were still using fonts (instead of text as curves), so that is also fixed.

**Comment by BigBlueHat** [comment]
@davidlehn and now we have a conflict in CHANGELOG.md which I'll be happy to let you fix. 😉

**Comment by BigBlueHat** [comment]
> @davidlehn and now we have a conflict in CHANGELOG.md which I'll be happy to let you fix. 😉

Nevermind. It's fixed. Let me know if you're happy with this so I an merge it. Cheers!

**Review by davidlehn — APPROVED** [review]

**Comment by davidlehn** [comment]
The "utopia-natcert" example JSON and SVG are more of a "demo" USA/DHS cert. Should they be updated to more clearly be for the fictitious "Utopia" somehow?

**Comment by BigBlueHat** [comment]
> The "utopia-natcert" example JSON and SVG are more of a "demo" USA/DHS cert. Should they be updated to more clearly be for the fictitious "Utopia" somehow?

Good call. I think this one at least needs a different folder and credential name. It also still has some data modeling issues around height... I'll remove it from this PR and create a separate one for that. We can also discuss whether this stays a demo of the USA/DHS cert or we make it a more general Utopia one. Thoughts on that, @msporny?

**Comment by BigBlueHat** [comment]
There's a Utopia-branded NatCert now. I'm realizing I need to do a pass later to sync up the issuer icons to match this new Utopia branding.

@jameseaster would love to sync up with how these look in your tests before we merge this one.

**Review by jameseaster — APPROVED** [review]
This all LGTM, I was able to run it locally without any issues! Will need to update configurations elsewhere as well as the following PR that includes queries once this is merged in.

**Review by msporny — APPROVED** [review]

**Pull request merged**